### ISSUE TITLE
Container that was on Deb 10, upgraded to 11, unable to update server binaries

### ISSUE THREAD

**Issue by Lusiphur** [Akkadius/akk-stack#19]
In file included from /home/eqemu/code/libs/perlbind/src/function.cpp:1:

```
/home/eqemu/code/libs/perlbind/include/perlbind/perlbind.h:30:10: fatal error: EXTERN.h: No such file or directory
   30 | #include <EXTERN.h>
      |          ^~~~~~~~~~
compilation terminated.
```

Seeing the previous error on an "update" command.

```
~/code/build$ cat /etc/debian_version
11.5
```

This is a container I probably haven't touched in about 6 months or so but I really don't want to lose all the config in it.

git pull seems clean.

```
eqemu@9d62640a6e11:~/code/build$ update
hint: Pulling without specifying how to reconcile divergent branches is
hint: discouraged. You can squelch this message by running one of the following
hint: commands sometime before your next pull:
hint:
hint:   git config pull.rebase false  # merge (the default strategy)
hint:   git config pull.rebase true   # rebase
hint:   git config pull.ff only       # fast-forward only
hint:
hint: You can replace "git config" with "git config --global" to set a default
hint: preference for all repositories. You can also pass --rebase, --no-rebase,
hint: or --ff-only on the command line to override the configured default per
hint: invocation.
remote: Enumerating objects: 10, done.
remote: Counting objects: 100% (10/10), done.
remote: Compressing objects: 100% (2/2), done.
remote: Total 10 (delta 8), reused 10 (delta 8), pack-reused 0
Unpacking objects: 100% (10/10), 2.42 KiB | 17.00 KiB/s, done.
From https://github.com/EQEmu/Server
 * [new branch]          bots/item_use_saylinks       -> origin/bots/item_use_saylinks
 * [new branch]          quest_api/event_zone_exports -> origin/quest_api/event_zone_exports
Already up to date.
```

**Comment by Lusiphur** [comment]
Followed the instructions in Discord to recreate the container .. to whit ..

```git pull && docker-compose pull eqemu-server && docker-compose kill eqemu-server && docker-compose up -d eqemu-server```

It did what it had to do (I had to re-add my hack to the Makefile as 'docker-compose' is now 'docker compose'.

I still can't build and now the server wont even start ....

At this point am I just taking a backup and blowing everything away and starting from scratch? Hate to lose all the fiddling I had to do with the login files and making sure certain rules were on.
